### DISCUSSION TITLE
ArchR Tutorial PBMC Data - Different Results

### DISCUSSION THREAD

**Discussion by brindhaprath** [GreenleafLab/ArchR#1982] [Questions / Documentation] [answered]
I've been analyzing the 10X Genomics PBMC ATAC-Seq data, following along with the "Brief Tutorial of ArchR" page to become familiar with the pipeline. However, I'm generating outputs that are different from the results on the website. I'm getting 12 clusters instead of 11, and the UMAP projections look different. Doublet filtering matches exactly, as does the number of nodes, but the number of edges is different. I've attached the code & outputs below.

Is this discrepancy simply due to a difference in updated packages? Is there much difference at all? If so, what does that mean in terms of interpretations?

![IterativeLSI ArchR Test Run](https://github.com/GreenleafLab/ArchR/assets/138097571/d7767d8f-8878-4b20-b975-66defcb31fb4)
![addClusters ArchR Test Run](https://github.com/GreenleafLab/ArchR/assets/138097571/22437817-2a47-4332-8199-b33088d367f1)
![addUMAP ArchR Test Run](https://github.com/GreenleafLab/ArchR/assets/138097571/26b67a16-1075-47ab-ac65-4f0bfa66332a)
![Plot-UMAP-Sample-Clusters](https://github.com/GreenleafLab/ArchR/assets/138097571/a57e304a-11ce-498f-baef-b28585b1e875)

**Comment by rcorces** [comment, marked as answer]
Yes I would imagine that this is due to changes in other packages. I guess it could also be due to differences in random seeds used.
